src/schema/show.ts

```diff
@@ -386,7 +413,38 @@ export const ShowType = new GraphQLObjectType({
       description: "The exhibition title",
       resolve: ({ name }) => (isExisty(name) ? name.trim() : name),
     },
+    nearbyShows: {
+      description: "Shows that are near (~75km) from this show",
+      type: showConnection,
+      args: pageable({
+        size: {
+          type: GraphQLInt,
+        },
+        sort: PartnerShowSorts,
+        status: EventStatus,
+      }),
+      resolve: (show, args, _context, { rootValue: { showsLoader } }) => {
+        // Bail with an empty array if we can't get the lat/long for this show
+        if (!show.location || !show.location.coordinates) {
+          return connectionFromArray([], args)
+        }
+        // OK, looks good
+        const coordinates = show.location.coordinates
+        const gravityOptions = {
+          ...args,
+          displayable: true,
+          near: `${coordinates.lat},${coordinates.lng}`,
+          max_distance: LOCAL_DISCOVERY_RADIUS_KM,
+        }
+
+        return showsLoader(gravityOptions).then(response => {
```

Comment:
I think `gravityOptions` might be missing the `parseRelayOptions` stuff that will convert from the `pageable` params (`first`, `after`, etc.) into `page` and `size` (to send to gravity).

Reply:
Great point! Moving this to WIP to add all that - thanks Matt!